boards/sodaq-explorer/include/periph_conf.h

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright (C)  2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_sodaq-explorer
+ * @{
+ *
+ * @file
+ * @brief       Configuration of CPU peripherals for the Sodaq LoRaWAN Explorer board
```

Comment:
nit picky, but consistent with other files here would be `Configuration of CPU peripherals for the SODAQ ExpLoRer board`